src/Ryujinx.Graphics.Nvdec.Vp9/Idct.cs

```diff
@@ -364,12 +359,11 @@ public static void HighbdIht8x864Add(ReadOnlySpan<int> input, Span<ushort> dest,
             }
         }
 
-        private static readonly HighbdTransform2D[] HighIht16 = new HighbdTransform2D[]
-        {
-            new HighbdTransform2D(HighbdIdct16, HighbdIdct16),   // DCT_DCT  = 0
-            new HighbdTransform2D(HighbdIadst16, HighbdIdct16),  // ADST_DCT = 1
-            new HighbdTransform2D(HighbdIdct16, HighbdIadst16),  // DCT_ADST = 2
-            new HighbdTransform2D(HighbdIadst16, HighbdIadst16)  // ADST_ADST = 3
+        private static readonly HighbdTransform2D[] _highIht16 = {
+            new(HighbdIdct16, HighbdIdct16),   // DCT_DCT  = 0
+            new(HighbdIadst16, HighbdIdct16),  // ADST_DCT = 1
+            new(HighbdIdct16, HighbdIadst16),  // DCT_ADST = 2
+            new(HighbdIadst16, HighbdIadst16), // ADST_ADST = 3
```

Comment:
```suggestion
            new(HighbdIdct16, HighbdIdct16),   // DCT_DCT  = 0
            new(HighbdIadst16, HighbdIdct16),  // ADST_DCT = 1
            new(HighbdIdct16, HighbdIadst16),  // DCT_ADST = 2
            new(HighbdIadst16, HighbdIadst16), // ADST_ADST = 3
```

Sometime comments are align, sometime not.